.travis.yml

```diff
@@ -56,8 +56,7 @@ script:
   case $JOB in
     emacs)
       # Check formatting (our formatting rules don’t work on Emacs 24).
-      # FIXME: this should run on Emacs 26 as well
-      if [[ $EMACS_VERSION == 25.* ]]; then
+      if [[ $EMACS_VERSION != 24.* ]]; then
```

Comment:
Fancy, this is beyond my own bash-fu.  Are Travis script guaranteed to run in bash, or is this POSIX-sh compliant too?

Reply:
Good question.  `!=` itself [may be POSIX](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128_05), but the wildcard may not be, and `[[ ]]` isn't, I think.  I can't find a definite source saying whether Travis uses bash (maybe dash?).  But it works as intended.

Reply:
> But it works as intended.

Let's merge :)